python/tvm/relay/op/contrib/tidl.py

```diff
@@ -391,11 +434,13 @@ def _max_pool_whitelist_fn(attrs, args):
                 and (strides[1] <= 2)
     return supported
 
-@tvm.ir.register_op_attr("multiply", "target.tidl")
-def _multiply_whitelist_fn(attrs, args):
-    #supported = True
-    supported = False
-    return supported
+@tvm.ir.register_op_attr("max", "target.tidl")
+def _max_whitelist_fn(attrs, args):
+    return True
+
+@tvm.ir.register_op_attr("mean", "target.tidl")
+def _mean_whitelist_fn(attrs, args):
```

Comment:
For max and mean, make sure we check for any attributes the converter doesn't support:
https://tvm.apache.org/docs/api/python/relay/index.html#tvm.relay.max
For example, we probably don't support when exclude is True and you may not support keepdims. It also isn't possible to support the case where axis is the channels_dim when mapping to pooling.

Reply:
I will add these checks. Thanks!

Reply:
TIDL supports "max" and "mean" for both keepdims being True and False.